dapps.asciidoc

```diff
@@ -164,16 +164,17 @@ Github link; https://github.com/embark-framework/embark
 
 Website link; https://github.com/embark-framework/embark
 
-[[dapp_develotment_tool_sec]
-==== DApp (development tool)
-DApp is a simple command line tool for smart contract development. It supports these common usecases:
+[[dapp_develotment_tool_sec]]
+==== Dapp (development tool)
```

Comment:
I thought that there was an explicit section for **dapp as a tool** in the [**dev-tools.asciidoc**](https://github.com/ethereumbook/ethereumbook/blob/develop/dev-tools.asciidoc) chapter, and that this chapter was for Dapp as an App not a tool.
It is also true that it appeared here before this PR. Perhaps @aantonop could give us some light on it.

Reply:
I agree that its unclear where the dev tools should go. I was just following the existing layout and figured it would get properly placed during the editing phase.